src/test/eo/org/eolang/math/number-tests.eo

```diff
@@ -385,6 +385,147 @@
       nop
     $.equal-to "0 cannot be raised to a negative power"
 
+[] > power-int-of-int-test
+  assert-that > @
+    number 2
+    .power 4
+    $.equal-to 16
+
+[] > power-int-of-float-test
+  assert-that > @
+    number 2.0
+    .power 4
+    $.equal-to 16.0
+
+[] > power-float-of-int-test
+  assert-that > @
+    number 2
+    .power 4.0
+    $.equal-to 16.0
+
+[] > power-float-of-float-test
+  assert-that > @
+    number 2.0
+    .power 4.0
+    $.equal-to 16.0
+
+[] > power-of-nan
+  eq. > @
+    ((number nan).power 2).as-bytes
+    nan.as-bytes
+
+[] > power-nan
+  eq. > @
+    ((number 10).power nan).as-bytes
+    nan.as-bytes
+
+[] > power-nan-of-nan
+  eq. > @
+    ((number nan).power nan).as-bytes
+    nan.as-bytes
+
+[] > power-is-int-zero-of-int
+  assert-that > @
+    number 2
+    .power 0
+    $.equal-to 1
+
+[] > power-is-float-zero-of-int
+  assert-that > @
+    number 2
+    .power 0.0
+    $.equal-to 1
```

Comment:
@maxonfjvipon Can you explain, why we return `int` here? In `float-power-of-int-zero` test we return `float` in such case.

Reply:
You are right. It should be float here

Reply:
@Graur it seems `eo` repository has not included [this](https://github.com/objectionary/eo/issues/942) to the release.  
 So this test won't work with `$.equal-to 1.0` until it's done

Reply:
@maxonfjvipon I believe it was released

Reply:
@Graur but why it fails [here](https://github.com/objectionary/eo-math/pull/41) then?

Reply:
@maxonfjvipon let me check it

Reply:
@Graur I ran it locally
<img width="420" alt="image" src="https://user-images.githubusercontent.com/37025995/186191262-34e7e9f6-3cd6-47a4-baf7-12a3a858f8d9.png">

Reply:
@maxonfjvipon Indeed, you are right! I can't see these changes in 0.27.1 version of EO

Reply:
@Graur could you please check if 0.27.2 version of EO contains needed changes?

Reply:
@maxonfjvipon You can check it yourself by simply investigating the "target" folder. I believe it should be there